### DISCUSSION TITLE
Hoppscotch's API to mange users and teams ?

### DISCUSSION THREAD

**Discussion by WestFarmer** [hoppscotch/hoppscotch#3580] [Q&A] [answered]
I searched  docs, not found.

Where is the API docs for the backend ?

**Comment by liyasthomas** [comment]
We have not yet exposed our backend server's API for operations, hence it's unavailable for general use.

If you think it would be a great feature for enterprise users feel free to [open a feature request](https://github.com/hoppscotch/hoppscotch/issues/new/choose) with a detailed description of the use case and how it would be beneficial for companies.

This is something we can implement in the enterprise edition if there are enough requests.

**Comment by liyasthomas** [comment, marked as answer]
Thanks for your patience and valuable feedback. This feature has been implemented in the latest release.

Here are the links to our documentation to learn more about:

- [APIs for User Management](https://docs.hoppscotch.io/documentation/self-host/enterprise-edition/admin-dashboard#apis-for-user-management)
- [APIs for Workspace Management](https://docs.hoppscotch.io/documentation/self-host/enterprise-edition/admin-dashboard#apis-for-workspace-management)
- [Infra tokens](https://docs.hoppscotch.io/documentation/self-host/enterprise-edition/admin-dashboard#infra-tokens)

Closing this ticket as this feature is now available in the recent version. Please feel free to reach out if you have any other concerns.
